### ISSUE TITLE
Should members who have left a project be listed as members in end-user APIs?

### ISSUE THREAD

**Issue by achou11** [digidem/mapeo-core-next#431]
## Description

We implemented `project.leave()` which updates a project member's role to `LEFT`. There's an open question as to whether listing these members when calling `MemberAPI.getMany()` should omit them by default (currently they're included). 

Gregor's potential suggestion is to omit them by default and include an option for the method e.g. `includeLeft` that would include them as an opt-in behavior.

Initial feeling is that we should implement this as described by the suggestion

**Comment by gmaclennan** [comment]
Yeah I'm not sure what's best, it depends on what feels most intuitive when using the API in the front-end. Open to this being whatever works best for @achou11 and @ErikSin. However if we are going to start adding a filter option to `getMany()` for other data types, it might start to be confusing if there is a "default" filter. On the other hand we already default to filtering out "deleted" records from `getMany()`.

**Comment by RangerMauve** [comment]
This is now more urgent since we have multi projects and project leaving.

**Comment by RangerMauve** [comment]
Related to the API I proposed for getting active members.

**Comment by achou11** [comment]
Part of me is wondering if it'd make sense for the filtering to be handled at the core-react level instead of here. Basically:

1. Core returns all historical members

2. In core react, we provide a way of filtering since it should be trivial to do that. Most likely a flag like `includeLeft` on the `useManyMembers()` read hook, but could also approach it differently e.g. a completely separate hook.

In retrospect, feels like we should've considered something similar for the `MapeoManager.listProjects()` API and how `useManyProjects()` works 😄

**Comment by achou11** [comment]
In terms of preference for filtering defaults, my initial thinking is that it'd make sense to roughly follow the same approach that the `MapeoManager.listProjects()` implements, meaning:

1. members who are considered invalid (e.g. left or blocked) are not included in the result.
2. there's some sort of flag that allows you to override this to get all historical members
